app/models/accounting/quickbooks/accounts_not_selected_error.rb

```diff
@@ -0,0 +1,6 @@
+module Accounting
+  module Quickbooks
+    class AccountsNotSelectedError < StandardError
```

Comment:
When testing to make sure that the StandardLoanDataExport is not affected by the qb connection or accounts settings, I realized this error should be raised for the UpdateAllLoanJob, so I slipped this in here.